docs/content/configuration/csi_ug_config_create_pvc.md

```diff
@@ -1,20 +1,20 @@
 # Creating a PersistentVolumeClaim (PVC)
 
-Create a PersistentVolumeClaim (PVC) yaml file for a persistent volume (PV).
+Create a PersistentVolumeClaim (PVC) YAML file for a persistent volume (PV).
 
 The IBM® block storage CSI driver supports using both file system and raw block volume types.
 
 **Important:** If not defined, the default type is `Filesystem`. Be sure to define the type as `Block` if this configuration is preferred.
 
 **Note:** The examples below create the PVC with a storage size 1 Gb. This can be changed, per customer needs.
 
-Use the sections below for creating yaml files for PVCs with file system and raw block volume types. After each yaml file creation, use the `kubectl apply` command.
+Use the sections below for creating YAML files for PVCs with file system and raw block volume types. After each YAML file creation, use the `kubectl apply` command.
```

Comment:
```suggestion
The IBM® block storage CSI driver supports using both file system and raw block volume modes.

**Important:** If not defined, the default mode is `Filesystem`. Be sure to define the mode as `Block` if this configuration is preferred.

**Note:** The examples below create the PVC with a storage size 1 Gb. This can be changed, per customer needs.

Use the sections below for creating YAML files for PVCs with file system and raw block volume modes. After each YAML file creation, use the `kubectl apply` command.
```

Reply:
DONE